libs/net/template/ReportExtensions.cs

```diff
@@ -558,12 +558,14 @@ public static bool IsPrivate(this ContentModel content, ReportEngineContentModel
             return spread.GroupBy(s => s.Key, s => s.Value);
         }
 
+        var excludeEmptyValues = settings?.ExcludeEmptyValues ?? false;
+
         var groups = groupBy switch
         {
             "mediaType" => content.GroupBy(c => c.MediaType?.Name ?? "Other").OrderBy(group => group.Key),
             "contentType" => content.GroupBy(c => c.ContentType.ToString()).OrderBy(group => group.Key),
-            "byline" => content.GroupBy(c => String.IsNullOrWhiteSpace(c.Byline) ? "Unknown" : c.Byline).OrderBy(group => group.Key),
-            "series" => content.GroupBy(c => c.SeriesId?.ToString() ?? "NA").OrderBy(group => group.Key),
+            "byline" => content.GroupBy(c => String.IsNullOrWhiteSpace(c.Byline) ? "Unknown" : c.Byline).Where((g) => excludeEmptyValues ? g.Key != "Unknown" : true).OrderBy(group => group.Key),
```

Comment:
It only works for byline and series currently.  Most of the other options are required values.